src/Carting.Api/Responses/V2/CartItem.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace Carting.Api.Responses.V2
+{
+
+    public class CartItem
+    {
+        /// <summary>
+        /// Id of the cart
+        /// </summary>
+        public string CartId { get; set; }
```

Comment:
This will add `CartId` to each Item, could you point me to a requirement of this?

Reply:
In this solution I didn't create a separate Cart object, as it has only a unique Id property, I'm storing it as a property of the CartItem.